package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "packages",
+  "version": "0.1.0",
+  "description": "Collection of JS modules and tools for WordPress development",
+  "repository": "git+https://github.com/WordPress/packages.git",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0+",
+  "keywords": [
+    "JavaScript",
+    "WordPress"
+  ],
+  "dependencies": {
+  },
+  "devDependencies": {
+    "babel-core": "^6.25.0",
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-preset-env": "^1.5.2",
+    "chalk": "^1.1.3",
+    "eslint": "~4.1.0",
+    "eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#d8abe35765088dcf138d8698cf5b0c4428d575cc",
+    "glob": "^7.1.2",
+    "lerna": "^2.0.0-rc.5",
+    "mkdirp": "^0.5.1"
+  },
+  "eslintConfig": {
```

Comment:
Curious why you included the eslint config here instead of a `.eslintrc`. More precisely, I wonder if IDEs can read the config from here?